README.md

```diff
@@ -370,3 +371,10 @@ Perform telescope.nvim functions in a selected directory
 - [Github](https://github.com/princejoogie/dir-telescope.nvim)
 - Author: [@princejoogie](https://github.com/princejoogie)
 - Twitter: [@princecaarlo](https://twitter.com/princecaarlo)
+
+## philippines-travel-level-map
```

Comment:
Hi, thanks for this. Please change the title to Philippines Travel Level Map

Reply:
Hello, I have changed the title. Thanks